onebusaway-android/src/google/java/org/onebusaway/android/map/googlemapsv2/BaseMapFragment.java

```diff
@@ -727,6 +729,20 @@ public void deactivate() {
         mListener = null;
     }
 
+    /**
+     * Adds a marker to the Map
+     *
+     * @param l is the marker's position in the map
+     * @return the added marker object,
+     * the marker object can be used to remove the marker from the map by calling marker.remove()
+     */
+    public Marker addMarker(Location l) {
```

Comment:
@barbeau yes, sorry. I couldn't see that when we assign the return result, we need to import the ```Marker```.